attributes/defaults.rb

```diff
@@ -73,7 +73,7 @@
 default['mysqld']['my.cnf']['mysqld']['log_error'] = '/var/log/mysql/error.log'
 default['mysqld']['my.cnf']['mysqld']['expire_logs_days'] = 10
 default['mysqld']['my.cnf']['mysqld']['max_binlog_size'] = '100M'
-default['mysqld']['my.cnf']['mysqld']['myisam-recover-options'] = 'BACKUP'
+default['mysqld']['my.cnf']['mysqld']['myisam_recover_options'] = 'BACKUP'
```

Comment:
I suppose it's definitly not an error, but I'd prefer sticking to the "use the defaults" policy. Can you remove that line from this commit?